dragonflybsd/testing-theory/dragonflybsd_regression_Q&A.md

```diff
@@ -39,16 +42,18 @@ contact maintainers for information.
 
 **What tests should be added?**
 
-All functionalities that will be added should have tests. More it will be
-possible to describe in detail after the implementation is completed.
+All functionalities that will be added should have tests. It will be
+possible to describe in more detail after the implementation is completed.
 
-What we would probably like to check:
+What we probably would  like to check:
 
 Values ​​from the ESRT table, however, a tool from
 [this site](https://reviews.freebsd.org/rG24f398e7a153a05a7e94ae8dd623e2b6d28d94eb)
 is needed. It seems that API will be compatible so one might work after updating
 EFI-specific including having the form like `dev/efi/efi.h`.
 
+Most propably for ESRT tables test we will use `kenv` tool.
```

Comment:
@PrzemyslawBanasiak13 however, `kenv` will not be used so you can remove that line

Reply:
Removed